xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -179,7 +179,7 @@ var _ httpfilter.ServerInterceptorBuilder = builder{}
 
 // BuildServerInterceptor is an optional interface builder implements in order
```

Comment:
Do you mind rewording this comment? It doesn't feel grammatically correct to me.

Reply:
`builder` in this case is a noun referencing the `builder` struct rather than reading as `an interface-builder`

Adding the word `that` between `interface` and `builder` to make it clear

Reply:
Thanks!